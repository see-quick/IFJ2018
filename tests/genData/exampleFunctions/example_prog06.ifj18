#count prime number

def primeNumber(a)
   
    divider = a - 1
    
    while divider > 1 do
    
        if ((a / divider) * divider) == a then
            a = 0
        else
        	divider = divider - 1
		end
    end
end

number = inputi
return_val = primeNumber(number)  

if return_val == 0 then
	print "\nNumber ", number, " is not a prime number\n"
else
	print "\nNumber ", number, " is a prime number\n"
end
