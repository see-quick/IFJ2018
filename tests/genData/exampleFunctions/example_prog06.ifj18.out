1
Number 1 is a prime number
